drivers/i2c/i2c_ite_enhance.c

```diff
@@ -1196,6 +1198,14 @@ static int i2c_enhance_init(const struct device *dev)
 				(IT8XXX2_SMB_SMB45CHS &= ~GENMASK(6, 4));
 		}
 
+		/* The property setting range of data hold time is 3-10. */
+		if (data_hold_time < 3) {
+			data_hold_time = 3;
```

Comment:
This check is not needed. The devicetree compiler will generate an error if the user specifies a `data-hold-time` outside the enum list.

Reply:
Removed and added defult value in yaml.